lib/module.js

```diff
@@ -32,8 +32,8 @@ module.exports = function nuxtAxios (_moduleOptions) {
   const prefix = process.env.API_PREFIX || moduleOptions.prefix || '/'
 
   // Apply defaults
-  const options = Object.assign(
-    {
+  const options = {
+    ...{
```

Comment:
remove spread nested object and use `{ optionOne: false, /* and so on */, ...moduleOptions }`